modules/installation-configuration-parameters.adoc

```diff
@@ -391,6 +391,10 @@ ifdef::osp[]
 |Additional security groups that are associated with compute machines.
 |A list of one or more UUIDs as strings. For example, `7ee219f3-d2e9-48a1-96c2-e7429f1b0da7`.
 
+|`compute.platform.openstack.zones`
+|{rh-openstack} Compute (Nova) availability zones (AZs) to install machines on. If this parameter is not set, the installer relies on the default settings for Nova that the {rh-openstack} administrator configured.
```

Comment:
I've seen `installation program` used more frequently in the installation docs than `installer`. I would assume consistency throughout the current doc would be more important, so that's maybe a topic to address later. But wanted to point it out.

Reply:
@codyhoag Yeah. I actually used "installation program" a lot during my first release at Red Hat, and was told to use "installer" on peer edits. No idea.

Reply:
Interesting :thinking:; Good to know. I've been trying to stay consistent with what I've seen, but perhaps I'm the one spreading around the wrong terminology.